docs/src/main/sphinx/language/overview.rst

```diff
@@ -0,0 +1,118 @@
+=====================
+SQL statement support
+=====================
+
+The SQL statement support in Trino can be categorized into a number of topics.
+Numerous statements are part of the core engine and therefore available in all
+use cases. On the other hand the details and architecture of the connected data
+sources prevent some SQL functionality.
+
+For example, if the data source does not support any write operations, Trino can
+not implement support for SQL statements like :doc:`/sql/delete`. Similarly if
+the underlying system does have any security concepts SQL statements like
+:doc:`/sql/create-role` can not provided by Trino and the connector.
+
+.. _sql-globally-available:
+
+Globally available statements
+-----------------------------
+
+The following statements are implemented in the core engine and available with
+any connector:
+
+* :doc:`/sql/call`
+* :doc:`/sql/deallocate-prepare`
+* :doc:`/sql/describe-input`
+* :doc:`/sql/describe-output`
+* :doc:`/sql/execute`
+* :doc:`/sql/explain`
+* :doc:`/sql/explain-analyze`
+* :doc:`/sql/prepare`
+* :doc:`/sql/reset-session`
+* :doc:`/sql/set-session`
+* :doc:`/sql/set-time-zone`
+* :doc:`/sql/show-functions`
+* :doc:`/sql/show-session`
+* :doc:`/sql/use`
```

Comment:
@electrum Should `VALUES` and `MATCH_RECOGNIZE` be part of this too? After all they are engine concepts and connectors don't participate in them.

Reply:
They are part of read operations .. since they are both part of SELECT statements and are listed in that section therefore. E.a .. no select support also means no match recognize support.

Reply:
Then we would want to document that connectors support order by, window functions, group by, etc.

Reply:
well... they are part of the same doc page for select and as such as linked. The values and match_recognize files show up as separate entities in the SQL syntax. If we drop linking to it we should follow up with doc updates on select, values and match_recognize to explain that they all work from the same select ..